### ISSUE TITLE
request for update

### ISSUE THREAD

**Issue by PipoDelgiudice** [aalises/age-of-empires-II-api#13]
You can update data for Age of empires II - definitive edition?

**Comment by JustusStorm** [comment]
Agreed. using this for custom database, an API is missing half of the civs in AEO2de.

**Comment by AndyTheNerd** [comment]
Hey guys, big fan of this API. Would love to see this updated with Age 2 DE content :)

**Comment by AndyTheNerd** [comment]
Please see this PR for updates on Age 2 civs. #14

**Comment by aalises** [comment]
Hello! thanks @AndyTheNerd for taking care of this! When I first created this it was a pet project to get myself around some Python and haven't had the time to go back at it much. It's great to see people using it :) if you feel that something is missing, feel free to add them in the `.csv`
